### ISSUE TITLE
Specify a MAC address for creating VMS

### ISSUE THREAD

**Issue by XzBuleSky** [retspen/webvirtcloud#469]
I want to specify a MAC address on the VM creation page and assign IP addresses in cooperation with DHCP

**Comment by catborise** [comment]
while creating instance with flavor option there is MAC adress input which you can change, but there arent other two options. we can add it.

**Comment by XzBuleSky** [comment]
@catborise  Hi, I also want to ask if webvirtcloud supports PXE startup?

**Comment by catborise** [comment]
i did add MAC address inputs for instance create methods

**Comment by catborise** [comment]
@XzBuleSky i think you want this for flavor method?

**Comment by XzBuleSky** [comment]
@aliceinwire Yes, I need the function of PXE to start the automatic installation system, but I can't find the corresponding option

**Comment by catborise** [comment]
@XzBuleSky after create the instance you can configure boot option. under settings->boot : boot order. select net then move up to make it first option. then run instance
